packages/p2p/src/components/my-ads/reduce-order-time-banner.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Please add a TODO to remove this component whenever it should be.

Reply:
Added the comment